rag_experiment_accelerator/config/environment.py

```diff
@@ -92,6 +92,7 @@ class Environment:
     azure_document_intelligence_endpoint: Optional[str]
     azure_document_intelligence_admin_key: Optional[str]
     azure_key_vault_endpoint: Optional[str]
+    pypdf_enabled: Optional[bool]
```

Comment:
It might be not the best place for the setting as it is not environment specific. Please, add it to the config structure instead (also to the config.schema.json). Maybe a new subsection or at least to the root (until we understand how to name the section)?